wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/de.md

```diff
@@ -76,7 +76,7 @@ Das Support-Team erwartet, dass Nutzer ihre vergangenen Verstöße auflisten und
 
 *Es wird vermutlich einige Zeit dauern, auf deine Beschwerde zu antworten, aber keine Beschwerden werden ignoriert.*
 
-## Was kann ich tun, wenn ich die Regeln gebrochen habe?
+## Was kann ich tun, wenn ich die Regeln gebrochen habe? {#was-kann-ich-tun-wenn-ich-die-Regeln-gebrochen-habe?}
```

Comment:
you still want it in lowercase

```suggestion
## Was kann ich tun, wenn ich die Regeln gebrochen habe? {#was-kann-ich-tun-wenn-ich-die-regeln-gebrochen-habe?}
```

Reply:
Yeah, right. Will change it in the other file too.